src/taskpane/pages/dashboard.tsx

```diff
@@ -1,46 +1,46 @@
-import {
-  IStackStyles,
-  DefaultPalette,
-  ISearchBoxStyles,
-  Stack,
-  SearchBox,
-  PrimaryButton,
-  ButtonType,
-} from "@fluentui/react";
-import React from "react";
-import Header from "../components/Header";
-/* global Word */
+import React, { useState } from "react";
+import data from "../../utils/data";
+import ReferenceList from "../components/ReferenceList";
+import SearchField from "../components/SearchField";
+// /* global Word */
 
-const stackStyles: IStackStyles = {
-  root: {
-    background: DefaultPalette.white,
-  },
-};
-const searchBoxStyles: Partial<ISearchBoxStyles> = { root: {} };
+function containsSearchTerm(keyword: string) {
+  return function (item) {
+    return [item.title, item.author, item.year].some((str) =>
+      str ? str.toLowerCase().includes(keyword.toLowerCase().trim()) : false
+    );
+  };
+}
+function onCheckboxChange(ev: React.FormEvent<HTMLElement | HTMLInputElement>) {
+  return function (item) {
+    if (item.title === ev.currentTarget.title) {
+      return { ...item, isChecked: !item.isChecked };
+    }
+    return item;
+  };
+}
 
 function Dashboard() {
-  const click = async () => {
-    return Word.run(async (context) => {
-      // insert a paragraph at the end of the document.
-      const paragraph = context.document.body.insertParagraph("Hello World!", Word.InsertLocation.end);
-      paragraph.font.color = "black";
-      paragraph.font.size = 30;
-      await context.sync();
+  const originalItems = data.map((item) => ({ ...item, isChecked: false }));
```

Comment:
Maybe rename `isChecked` to `isSelected` to better reflect the purpose instead of the implementation?